tests/test_nested_suites.c

```diff
@@ -0,0 +1,157 @@
+#include "munit.h"
+
+typedef struct {
+    int suite_setup_calls;
+    int suite_teardown_calls;
+    int test_setup_calls;
+    int test_teardown_calls;
+    int test_calls_a;
+    int test_calls_b;
+} test_call_t;
+
+static void reset_test(test_call_t *tests) {
+    tests->suite_setup_calls = 0;
+    tests->suite_teardown_calls = 0;
+    tests->test_setup_calls = 0;
+    tests->test_teardown_calls = 0;
+    tests->test_calls_a = 0;
+    tests->test_calls_b = 0;
+}
+
+static void assert_calls(test_call_t *tests, int suite_setup_calls, int suite_teardown_calls, int test_setup_calls,
+                         int test_teardown_calls, int test_calls_a, int test_calls_b) {
+    munit_assert(suite_setup_calls == tests->suite_setup_calls);
```

Comment:
Is it possible to use `munit_assert_int(suite_setup_calls, ==, tests->suite_setup_calls)` ?